### DISCUSSION TITLE
$3 = 5 (Step direction invert, mask)

### DISCUSSION THREAD

**Discussion by quotelawrence** [arkypita/LaserGRBL#941] [Q&A]
after using the modifications to the settings $3=7, and the lettering and directions 
are all correct if I hit the home button the machine now travels so far 
in the wrong direction I get a travel arrest and the machine shuts down
is there another spot where I need to change a setting to stop the 
machine from going to a position that does not exist, ?

**Comment by gmmanonymus111** [comment]
[GRBL homing setup](https://github.com/gnea/grbl/wiki/Set-up-the-Homing-Cycle#homing-direction) might be helpful for you

**Comment by quotelawrence** [comment]
ok thanks I am still very ignorant of the different functions and setting so I noted i used $3 = 7, and the table took off in the y direction opposite as if the zero (home was in the negative area,) does that make any sense I will experiment I just hate crashing the machine over my ignorance
